packages/amplify-graphql-api-construct/src/model-datasource-strategy-types.ts

```diff
@@ -132,7 +132,7 @@ export type SqlModelDataSourceDbConnectionConfig =
  */
 export interface SqlModelDataSourceSsmDbConnectionStringConfig {
   /** The SSM Path to the secure connection string used for connecting to the database. **/
```

Comment:
Add additional documentation for the resolution strategy if multiple values are specified
```suggestion
  /**
   * The SSM Path to the secure connection string used for connecting to the database. If more than one path is provided,
   * the SQL Lambda will attempt to retrieve connection information from each path in order until it finds a valid 
   * path entry, then stop. If the connection information contained in that path is invalid, the SQL Lambda will not 
   * attempt to retrieve connection information from subsequent paths in the array.
   **/
```